eBay-brand-tracker/ebay_bt_venv/Lib/site-packages/_distutils_hack/__init__.py

```
@@ -0,0 +1,239 @@
+        del sys.modules[name]
+
+
+def enabled():
+    """
+    Allow selection of distutils by environment variable.
+    """
+    which = os.environ.get('SETUPTOOLS_USE_DISTUTILS', 'local')
+    if which == 'stdlib':
+        import warnings
+
+        warnings.warn(
```

Ruff

No explicit `stacklevel` keyword argument found

(B028)